src/metabase/util/fonts.clj

```diff
@@ -0,0 +1,41 @@
+(ns metabase.util.fonts
+  "font loading functionality."
+  (:require [clojure.string :as str]
+            [clojure.tools.logging :as log]
+            [metabase.util.files :as u.files]))
+
+(def font-path
+  "font resources path as a String."
+  "./resources/frontend_client/app/fonts/")
+
+(defn- normalize-font-dirname
+  "Use a font's directory to derive a Display Name.
+
+  Underscores become spaces, and then split on dashes and take the first word.
+  This is done because of the Lato-v16-latin directory name."
+  [dirname]
+  (-> dirname
+      (str/split #"-")
+      first
+      (str/replace #"_" " ")))
+
+(defn- contains-font-file?
+  [path]
+  ;; todo: expand this to allow other font formats?
+  (some #(str/includes? % ".woff") (u.files/files-seq path)))
```

Comment:
Shouldn't `str/includes?` really be `str/ends-with?`?

Reply:
I did this intentionally since the files are actually `.woff2`, but wanted to also allow `.woff`. 
I could be more strict and do `(str/ends-with? ".woff2")`

Reply:
I see. I would match with a regex then.